### DISCUSSION TITLE
New phone with Andoid 13. Cannot start Decom G7 sensor.

### DISCUSSION THREAD

**Discussion by bertroode** [NightscoutFoundation/xDrip#2857] [Q&A]
On my new Samsung A54 with Android 13 and the latest nightly of Xdrip+ I cannot start a new Dexcom sensor.
I get stuck when a message appears again and again about the need of a background location for Android 10 and higher.
I reported this problem a few weeks ago in the app but so far it still exists.

Regards,
Bert Roode

**Comment by Navid200** [comment]
Please go to the page on the following link.
https://navid200.github.io/xDrip/docs/Dexcom-Basics.html

Scroll down to get to the heading "Location and Bluetooth".
Follow the instructions under that heading.

**Comment by bertroode** [comment]
I adapted the settings but the procedure differs from what is described on the Dexcom page.
Could it be possible to add some code in xdrip+ to set this permission the same way as the other permissions?

Regards,
Bert Roode

**Reply by Navid200** [reply]
No, this is not an xDrip setting.
It is an Android setting.

**Reply by Navid200** [reply]
Can you please show a screenshot of what you see that is different than what you expect?

**Reply by oracess** [reply]
Have the same problem with A53, A52, A51 Android 13 and nightly d46b342-2023.06.17.
Reference of the link above is for Andoid 12 and not 13. I found all things and points on Android 13 but it is not pairing until now. Workaraund is only pairing with dexcom app, then uninstall completly dexcom app and restart phone. Now bluetooth connection is OK and G7 works with xdrip+ nightly d46b342-2023.06.17 !! 
![Screenshot_20230619_114448_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/41123950/1e7db4da-836d-4238-a239-365799859bbe)


![Screenshot_20230619_115454_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/41123950/aebe71a4-0ecd-4ad5-b632-b1b483c320e4)

**Comment by Navid200** [comment]
@oracess 
> Reference of the link above is for Andoid 12 and not 13.

What do you mean by that statement?
This is Android 13:
![Screenshot_20230619-082320](https://github.com/NightscoutFoundation/xDrip/assets/51497406/68ff1442-620b-4283-8e67-d84d0f87c436)

**Comment by oracess** [comment]
As i wrote all was found for Android 13, but it does not pair the Sensor g7 to Bluetooth of A53. Dexcom or BYODA make it! So there must be an error by xdrip+ or handling with xdrip+ nightly. Look at my pictures - BLUETOOTH NOT CONNECTED - DATA RECEIVED!

**Reply by Navid200** [reply]
I would like to help.  
But, I take it you prefer someone else to help.  So, I will stay out of this thread.
